tests/jobs/test_models.py

```diff
@@ -146,7 +147,7 @@ def test_sync_terminated(request, mocker, sparkjob_provisioner_mocks,
         'atmo.clusters.provisioners.ClusterProvisioner.info',
         return_value={
             'creation_datetime': one_hour_ago,
-            'ready_datetime': None,
+            'ready_datetime': one_hour_ago,
```

Comment:
Huh, why this change?

Reply:
This is for the test for recording the "sparkjob-run-time" which calculates run time by using `end_datetime` minus `ready_datetime`. Also, it doesn't make sense to have a end date w/o a ready date.

Reply:
Got it.